### ISSUE TITLE
Accessibility compliance: select element missing an associated label

### ISSUE THREAD

**Issue by ivanteoh** [eea/eea.facetednavigation#30]
Add a label element that surrounds the control's label. Set the for attribute on the label element to the same value as the idattribute of the control. And/or add a title attribute to the input element. And/or create a label element that contains the inputelement.

For example, the language select input element does not have label close to it, refer to screen shot. 

Refer to [Web accessibility checker 91](http://achecker.ca/checker/suggestion.php?id=91)

![Resources MHCS - Google Chrome_267](https://f.cloud.github.com/assets/900434/359982/84c539f6-a178-11e2-929f-4e55cfb11c77.png)

Thank you
